shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -0,0 +1,72 @@
+use cassandra_cpp::*;
```

Comment:
What are we importing from `cassandra_cpp`? I think being explicit would be better.

Reply:
Has this been addressed in a commit you haven't pushed yet?

Reply:
this is addressed now